service/lib/agama/storage/config_conversions/to_json_conversions/search.rb

```diff
@@ -50,6 +59,14 @@ def convert_condition
 
             { name: name }
           end
+
+          # Whether the search can be aliased with the special "*" string
+          def anything?(search_hash)
```

Comment:
What about adding `Configs::Search#anything?` instead of comparing the hashes?

Reply:
Done (with another method name)